release_dashboard/test/fakes/fake_start_context.dart

```diff
@@ -53,6 +54,7 @@ class FakeStartContext extends ReleaseDashboardStartContext {
       processManager: processManager,
       stdio: stdio,
     );
+    syncStatusWithState ??= () {};
```

Comment:
The fake does not have concepts of state. Therefore, it does not need to sync status with state.